### DISCUSSION TITLE
A way to avoid the Imgur upload step?

### DISCUSSION THREAD

**Discussion by FrostMiranda** [JoePenna/Dreambooth-Stable-Diffusion#107] [General]
I find this step so labor intensive. Getting the URLs, pasting them one by one, formatting them in the notebook. Is there a way around this?  Or at least a way to get the URLs automatically from Imgur?

**Comment by d1mski** [comment]
Create a trained_images folder manually and drop your training photos in the folder. Seems to work for me skipping those two image processes.

**Reply by FrostMiranda** [reply]
You’re absolutely right. This didn’t work the first time I tried it because my images weren’t PNGs. But once I converted them, uploading them to training_images was all it took.
